README.md

```diff
@@ -0,0 +1,116 @@
+# Math Magician
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+    - [🛠 Built With](#built-with)
+        - [Tech Stack](#tech-stack)
+        - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+    - [Setup](#setup)
+    - [Prerequisites](#prerequisites)
+    - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math Magician <a name="about-project"></a>
+
+**Math Magician** This is online calculator built with React
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Programming languages</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">React</a></li>
+  </ul>
+ </details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+    <li><a href="https://github.com/">Github</a></li>
+    <li><a href="https://eslint.org/">Linters</a></li>
+    <li><a href="https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow">Gitflow</a></li>
+  </ul>
+ </details>
+ <details>
+  <summary>Frameworks</summary>
+  <ul>
+    <li><a href="https://getbootstrap.com/">Bootstrap 5</a></li>
+  </ul>
+ </details>
+<details>
+  <summary>Client</summary>
+ </details>
+
+<details>
+  <summary>Server</summary>
+</details>
+
+<details>
+<summary>Database</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Math Magician Project Setup**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Live Demo <a name="live-demo"></a>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/edenlisk/math-magician-react.git
+```
+
+### Deployment
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Edenlisk**
+
+- GitHub: [@Edenlisk](https://github.com/edenlisk)
+- Twitter: [@Edenlisk](https://twitter.com/nkumbuyedeni)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Please consider giving a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly make sure that you have all the main headers from [this README tempate](https://github.com/microverseinc/readme-template) present in your repo. The only optional section is FAQ. Also kindly make sure you have added the section `Future FEatures`, including 1-3 potential features to be added.  These are part of the general requirements for every project you will be working on here at Microverse.